src/modules/account/components/BackupAccount/BackupAccount.js

```diff
@@ -0,0 +1,46 @@
+import React, { useRef, useState } from 'react';
+import { withRouter } from 'react-router';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import MultiStep from 'src/modules/common/components/MultiStep';
+import EnterPasswordForm from '@auth/components/EnterPasswordForm';
+import SavePassphrase from '@auth/components/SavePassphrase/SavePassphrase';
+import ConfirmPassphrase from '@auth/components/ConfirmPassphrase/confirmPassphrase';
+import SetPasswordSuccess from '@auth/components/setPasswordSuccess';
+import { useCurrentAccount } from '@account/hooks/useCurrentAccount';
+import styles from './BackupAccount.css';
+
+const BackupAccount = ({ history }) => {
+  const multiStepRef = useRef(null);
+  const [account] = useCurrentAccount();
+  const [passphrase, setPassphrase] = useState('');
+
+  const onEnterPasswordSuccess = ({ recoveryPhrase }) => {
+    setPassphrase(recoveryPhrase);
+    multiStepRef.current.next();
+  };
+
+  const onComplete = () => {
+    history.push('/wallet');
```

Comment:
please reuse the route not hardcoded value 
packages/views/screens/router/routes.js
`route.wallets.path`